.github/workflows/swiftshader.yml

```diff
@@ -0,0 +1,40 @@
+name: SwiftShader
+
+on: push
+# TODO: replace the above line with manual trigger as follows.
+# on:
+#   workflow_dispatch:
+
+jobs:
+  swiftshader:
+    name: Test Filament + Vulkan + SwiftShader
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@v2.0.0
```

Comment:
Do we still need to checkout Filament?

Reply:
This ties in with your previous question a little; the docker run command uses `-v` to access the host copy of Filament, which would be the branch that's being pushed.  I think the checkout action is necessary to give the host access to this.

Reply:
Got it, sounds good